data/json/items/gun/shot.json

```diff
@@ -89,7 +88,10 @@
         "volume": 1,
         "price": 5000
     },
-    "burst": 2,
+    "modes": [
+      [ "DEFAULT", "single", 1 ],
+      [ "AUTO", "double", 2 ]
```

Comment:
`AUTO` is used to handle the burst fire key and by NPC's. I'd like to deprecate the former and rework the latter so NPC's better understand their options. For now though that mode is probably best named `DOUBLE` as it's not automatic fire.